### ISSUE TITLE
Resource priority amidst contention: async image pull or on-demand fetch

### ISSUE THREAD

**Issue by ganeshkumar5699** [containerd/accelerated-container-image#228]
If during async background image pull, another pod starts or has an on-demand image fetch call, is the resource priority (for IO, CPU) higher for the async pull process or the on-demand fetch call?

**Comment by lihuiba** [comment]
I don't think we have QoS or priority control at present. All we have done is to throttle bandwidth of background downloading.

**Comment by lihuiba** [comment]
I don't think we have QoS or priority control at present. All we have done is to throttle the bandwidth of background downloading. As throttling can be taken as a basic form of (lowering) priority, what is your opinion in the implementation perspective view?
